data/constants.json

```diff
@@ -65,7 +66,7 @@
   "strategusEquivalentDistance": 0.2, /* Distance at which two points are consider equivalent. */
   "strategusInteractionDistance": 1, /* Distance to attack, trade, ... */
   "strategusViewDistance": 8,
-  "strategusSpawningPositionCenter": [122, -105],
+  "strategusSpawningPositionCenter": [ 122, -105 ],

```

Comment:
To discard

Reply:
You reviewed it before i ask for the review :D 
I caught it before!
![image](https://github.com/verdie-g/crpg/assets/47090987/196dbed5-ee0d-4fab-b808-0ecade522c2f)